docker/Dockerfile.prod

```diff
@@ -7,7 +7,6 @@ RUN tar -xvjf /tmp/*.tar.bz2 -C /var/www/html/ && \
 COPY config/packages/parameters.yaml.dist config/packages/parameters.yaml
 COPY config/packages/prod/monolog.yaml.docker config/packages/prod/monolog.yaml
 COPY .env.docker .env
```

Comment:
This can be removed right? As I see it, the env var is not evaluated via this file any longer. But should be placed in the ENV by docker compose or another means.